stories/svgs/utils.js

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
Created this to prevent duplication and centralise to avoid future duplicate effort to change (if needed)